tests/e2e/testfiles/scenarios.yaml

```diff
@@ -1463,9 +1463,8 @@
     # Last steps perform Clean UP
     - "Restore template [testfiles/powerflex-templates/powerflex-secret-template.yaml] for [pflex]"
     - "Restore template [testfiles/powerflex-templates/powerflex-storageclass-template.yaml] for [pflex]"
-    - "Restore template [testfiles/application-mobility-templates/csm_application_mobility_no_velero.yaml] for [application-mobility]"
+    - "Restore template [testfiles/application-mobility-templates/csm_application_mobility_with_pflex.yaml] for [application-mobility]"
```

Comment:
Are these changes intentional and needed in this PR ? They seem unrelated to Powermax e2e tests.

Reply:
The scenarios weren't ready while creating the draft. changed now.